app/main.py

```diff
@@ -16,4 +16,27 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+variables_names_array = [
+    lucky_number,
+    pi,
+    one_is_a_prime_number,
+    name,
+    my_favourite_films,
+    profile_info,
+    marks,
+    collection_of_coins
+]
+
+sorted_variables = {
```

Comment:
No calculations in this task - simply pass the variables in the correct list in dict